sonyapilib/device.py

```diff
@@ -740,7 +743,9 @@ def send_authentication(self, pin):
 
     def wakeonlan(self, broadcast=None):
         """Start the device via wakeonlan."""
-        broadcast = broadcast or '255.255.255.255'
+        broadcast = broadcast or (self.broadcast_address
+                                  if hasattr(self, 'broadcast_address')
```

Comment:
the hasattr check isn't really necessary as it's always set in init?

Reply:
I added this to prevent errors when loading old json file which doesn't have broadcast address stored yet

Reply:
Similar to the other MR, there should probably a test case which creates a device from an 'old' and 'new' json

Reply:
Will add test for broadcast address after merging the other MR so i will not add duplicate commits

Reply:
Done

Reply:
Please let me resolve the threads next time. It is easier to review :)